pkg/reconciler/labeler/labeler_test.go

```diff
@@ -57,22 +58,43 @@ func TestReconcile(t *testing.T) {
 		Objects: []runtime.Object{
 			simpleRunLatest("default", "first-reconcile", "the-config"),
 			simpleConfig("default", "the-config"),
-			simpleRevision("default", "the-config"),
+			rev("default", "the-config"),
 		},
 		WantPatches: []clientgotesting.PatchActionImpl{
+			patchAddLabel("default", rev("default", "the-config").Name,
+				"serving.knative.dev/route", "first-reconcile", "v1"),
 			patchAddLabel("default", "the-config", "serving.knative.dev/route", "first-reconcile", "v1"),
 		},
 		Key: "default/first-reconcile",
 	}, {
 		Name: "steady state",
 		Objects: []runtime.Object{
 			simpleRunLatest("default", "steady-state", "the-config"),
-			routeLabel(simpleConfig("default", "the-config"), "steady-state"),
-			simpleRevision("default", "the-config"),
+			simpleConfig("default", "the-config",
```

Comment:
Why change `simpleRevision` -> `rev`, but leave `simpleConfig` alone?

Reply:
The rename was because @vagababov complained that I was making `simpleRevision` not simple by adding options.  `simpleConfig` is still simple.  I don't care, I can rename things if you want 🙄